src/Document/Content.php

```diff
@@ -42,7 +42,7 @@ class Content
     protected $name;
 
     /**
-     * @PHPCRODM\String
+     * @PHPCRODM\Field(type="string")
```

Comment:
it's here for BC reasons and going to be removed in 2.0 (next version will be 1.3)